docs/geedocs/5.3.0/answer/4492617.html

```diff
@@ -31,9 +31,8 @@ <h2>Uninstalling</h2>
 <code>/etc/init.d/geserver stop</code>
 </p>
 </li>
-<li>Navigate to the location at which you initially cloned the GEE git repo.</li>
-<li>Within the GEE git repo, navigate to <code>earthenterprise/earth_enterprise/src/installer</code>
-and run the <code>uninstall_fusion.sh</code> and <code>uninstall_server.sh</code> scripts.</li></ol></div>
+<li>Navigate to the location at which you initially cloned the GEE git repo or to the installation location (<code>/opt/google</code>).</li>
```

Comment:
At the page footer let's add `©2019 Open GEE Contributors` to the copyright notice be: `©2015 Google, ©2019 Open GEE Contributors)`

Reply:
done